apps/web/src/modules/create-proposal/components/TransactionForm/Migration/Migration.schema.ts

```diff
@@ -0,0 +1,49 @@
+import * as yup from 'yup'
+
+import { addressValidationSchema } from 'src/utils/yup'
+
+// WILL NEED DIFFERENT VALIDATIONS FOR DIFFERENT SECTIONS
+
+export interface MigrationFormValues {
```

Comment:
guessing this was left in from copy pasting the create NFT proposal template is this still needed?

Reply:
yup, removing this file